src/components/CourseCard/__test__/CourseCard.test.jsx

```diff
@@ -48,4 +48,4 @@ describe('Course Component test', () => {
       .toJSON();
     expect(tree).toMatchSnapshot();
   });
-});
+});
```

Comment:
```suggestion
});

```
This change does not seem to be required